sdks/wasm/sdk/Templates/DotNet/.template.config/template.json

```diff
@@ -0,0 +1,19 @@
+{
+    "author": "Mono",
+    "classifications": [ "Mono/WebAssembly" ],
+    "name": "Mono WebAssembly Starter",
+    "identity": "WasmTemplate.1",        
```

Comment:
is the 1 significant in any way?

Reply:
No.  I was just following the naming convention suggestion doc.

Reply:
https://github.com/dotnet/templating/wiki/Naming-and-default-value-forms